### PR TITLE
Updated Polish translation to 100% string coverage

### PR DESCRIPTION
**Pull request by zutmkr**
All the English strings are translated, it should be a good base for spelling/gramma checks. I will iterate through it some time more after I will play the game more and catch some weird sounding phrases/words.

### CODE REVIEW

**Comment by AJenbo** [comment]
From a technical standpoint this look good. @qndel can you handle the actual review of this :D

File: Translations/pl.po [hunk 1]
```diff
@@ -1272,300 +1251,241 @@ msgid "while in stores"
...
 msgid "Belt item {}"
-msgstr "Sprzedaż"
+msgstr "Element pasa {}"
 
 #: Source/diablo.cpp:1440
-#, fuzzy
-#| msgid "1 - 8: Use Belt item"
 msgid "Use Belt item."
-msgstr "1 - 8: Sięgnięcie po przedmiot z pasa"
+msgstr "Sięgnięcie po przedmiot z pasa."
```
Comment (qndel) [inline Translations/pl.po:1259]:
```suggestion
msgstr "Użycie przedmiotu z pasa."
```

File: Translations/pl.po [hunk 2]
```diff
@@ -1272,300 +1251,241 @@ msgid "while in stores"
...
 msgid "Quick save"
-msgstr ""
+msgstr "Szybki zapis"
 
 #: Source/diablo.cpp:1483
-#, fuzzy
-#| msgid "Savegame:"
 msgid "Saves the game."
-msgstr "Zapis Gry:"
+msgstr "Zapisuje Grę."
```
Comment (qndel) [inline Translations/pl.po:1283]:
```suggestion
msgstr "Zapisuje grę."
```

File: Translations/pl.po [hunk 3]
```diff
@@ -1272,300 +1251,241 @@ msgid "while in stores"
...
 msgid "Pauses the game."
-msgstr ""
+msgstr "Wstrzymuje grę."
 
 #: Source/diablo.cpp:1618
-#, fuzzy
-#| msgid "increase magic"
 msgid "Decrease Gamma"
-msgstr "zwiększa magię"
+msgstr "Zmniejsza Gamme"
```
Comment (qndel) [inline Translations/pl.po:1395]:
```suggestion
msgstr "Zmniejsz jasność"
```

File: Translations/pl.po [hunk 4]
```diff
@@ -1272,300 +1251,241 @@ msgid "while in stores"
...
 msgid "Reduce screen brightness."
-msgstr "F: Zmniejsza jasność ekranu"
+msgstr "Zmniejsza jasność ekranu."
 
 #: Source/diablo.cpp:1626
-#, fuzzy
-#| msgid "increase magic"
 msgid "Increase Gamma"
-msgstr "zwiększa magię"
+msgstr "Zwiększ Gamme"
```
Comment (qndel) [inline Translations/pl.po:1403]:
```suggestion
msgstr "Zwiększ jasność"
```

File: Translations/pl.po [hunk 5]
```diff
@@ -2009,10 +1929,8 @@ msgstr ""
 "nie Prawym Przyciskiem Myszy."
 
 #: Source/help.cpp:69
-#, fuzzy
-#| msgid "$Gold"
 msgid "$Gold:"
-msgstr "$Złota"
+msgstr "$Złota:"
```
Comment (qndel) [inline Translations/pl.po:1933]:
```suggestion
msgstr "$Złoto:"
```

File: Translations/pl.po [hunk 6]
```diff
@@ -6188,646 +6090,642 @@ msgstr "{:s} (wył.)"
...
-#| msgctxt "spell"
-#| msgid "Flash"
 msgid "Splash"
-msgstr "Rozbłysk"
+msgstr "Rozprysk"
 
 #: Source/options.cpp:559
 msgid "Shown splash screen."
-msgstr ""
+msgstr "Pokaż splash screen."
```
Comment (qndel) [inline Translations/pl.po:6141]:
```suggestion
msgstr "Pokaż ekran powitalny."
```

File: Translations/pl.po [hunk 7]
```diff
@@ -6188,646 +6090,642 @@ msgstr "{:s} (wył.)"
...
-msgstr ""
+msgstr "Pokaż intro."
 
 #: Source/options.cpp:559
-#, fuzzy
-#| msgctxt "spell"
-#| msgid "Flash"
 msgid "Splash"
-msgstr "Rozbłysk"
+msgstr "Rozprysk"
```
Comment (qndel) [inline Translations/pl.po:6137]:
```suggestion
msgstr "Powitanie"
```

File: Translations/pl.po [hunk 8]
```diff
@@ -6188,646 +6090,642 @@ msgstr "{:s} (wył.)"
...
 
 #: Source/options.cpp:613
 msgid "Auto Equip Sound"
-msgstr ""
+msgstr "Auto Wyposażenie Dźwięk"
 
 #: Source/options.cpp:613
 msgid "Automatically equipping items on pickup emits the equipment sound."
 msgstr ""
+"Automatyczne wyposażanie przedmiotów przy podnoszeniu powoduje emisję "
```
Comment (qndel) [inline Translations/pl.po:6182]:
```suggestion
"Automatyczne zakładanie przedmiotów przy podnoszeniu powoduje emisję "
```

File: Translations/pl.po [hunk 9]
```diff
@@ -6188,646 +6090,642 @@ msgstr "{:s} (wył.)"
...
 
 #: Source/options.cpp:746
 msgid "Nearest Pixel"
-msgstr ""
+msgstr "Najbliższy piksel"
 
 #: Source/options.cpp:747
 msgid "Bilinear"
-msgstr ""
+msgstr "Bilinearny"
```
Comment (qndel) [inline Translations/pl.po:6296]:
```suggestion
msgstr "Bilinearne"
```

File: Translations/pl.po [hunk 10]
```diff
@@ -6188,646 +6090,642 @@ msgstr "{:s} (wył.)"
...
 msgstr ""
+"Umożliwia skalowanie obrazu z rozdzielczości gry do rozdzielczości monitora. "
+"Zapobiega zmianie rozdzielczości monitora i umożliwia zmianę rozmiaru okna."
 
 #: Source/options.cpp:744
-#, fuzzy
-#| msgid "high durability"
 msgid "Scaling Quality"
-msgstr "wysoka wytrzymałość"
+msgstr "Jakość skalowania"
```
Comment (qndel) [inline Translations/pl.po:6283]:
```suggestion
msgstr "Rodzaj skalowania"
```

File: Translations/pl.po [hunk 11]
```diff
@@ -6188,646 +6090,642 @@ msgstr "{:s} (wył.)"
...
 
 #: Source/options.cpp:747
 msgid "Bilinear"
-msgstr ""
+msgstr "Bilinearny"
 
 #: Source/options.cpp:748
 msgid "Anisotropic"
-msgstr ""
+msgstr "Anizotropowy"
```
Comment (qndel) [inline Translations/pl.po:6300]:
```suggestion
msgstr "Anizotropowe"
```

File: Translations/pl.po [hunk 12]
```diff
@@ -6188,646 +6090,642 @@ msgstr "{:s} (wył.)"
...
 "frame. Disabling it can help with mouse lag on some systems."
 msgstr ""
+"Wymusza oczekiwanie na synchronizację pionową. Zapobiega efektowi rozrywania "
+"podczas rysowania klatki. Wyłączenie tej funkcji może pomóc w eliminacji "
+"opóźnień myszy w niektórych systemach."
 
 #: Source/options.cpp:754
 msgid "Color Cycling"
-msgstr ""
+msgstr "Przełącznie kolorów"
```
Comment (qndel) [inline Translations/pl.po:6325]:
```suggestion
msgstr "Przełączanie kolorów"
```

File: Translations/pl.po [hunk 13]
```diff
@@ -6188,646 +6090,642 @@ msgstr "{:s} (wył.)"
...
 
 #: Source/options.cpp:757
 msgid "Use a hardware cursor"
-msgstr ""
+msgstr "Użyj kursora sprzętowego"
 
 #: Source/options.cpp:758
 msgid "Hardware Cursor For Items"
-msgstr ""
+msgstr "Kursor sprzętowy dla elementów"
```
Comment (qndel) [inline Translations/pl.po:6351]:
```suggestion
msgstr "Kursor sprzętowy dla przedmiotów"
```

File: Translations/pl.po [hunk 14]
```diff
@@ -6188,646 +6090,642 @@ msgstr "{:s} (wył.)"
...
 
 #: Source/options.cpp:758
 msgid "Use a hardware cursor for items."
-msgstr ""
+msgstr "Użyj kursora sprzętowego dla elementów."
```
Comment (qndel) [inline Translations/pl.po:6355]:
```suggestion
msgstr "Użyj kursora sprzętowego dla przedmiotów."
```

File: Translations/pl.po [hunk 15]
```diff
@@ -6188,646 +6090,642 @@ msgstr "{:s} (wył.)"
...
 
 #: Source/options.cpp:762
 msgid "Displays the FPS in the upper left corner of the screen."
-msgstr ""
+msgstr "Wyświetlanie liczby FPS w lewym górnym rogu ekranu."
 
 #: Source/options.cpp:763
 msgid "Show health values"
-msgstr ""
+msgstr "Pokaż wartości Zdrowia"
```
Comment (qndel) [inline Translations/pl.po:6389]:
```suggestion
msgstr "Pokaż ilość życia"
```

File: Translations/pl.po [hunk 16]
```diff
@@ -6188,646 +6090,642 @@ msgstr "{:s} (wył.)"
...
 
 #: Source/options.cpp:763
 msgid "Displays current / max health value on health globe."
-msgstr ""
+msgstr "Wyświetla aktualną / maksymalną wartość stanu zdrowia na kuli zdrowia."
 
 #: Source/options.cpp:764
 msgid "Show mana values"
-msgstr ""
+msgstr "Pokaż wartości many"
```
Comment (qndel) [inline Translations/pl.po:6397]:
```suggestion
msgstr "Pokaż ilość many"
```

File: Translations/pl.po [hunk 17]
```diff
@@ -6188,646 +6090,642 @@ msgstr "{:s} (wył.)"
...
-msgstr "Kontynuuj rozgrywkę"
+msgstr "Bieg w mieście"
 
 #: Source/options.cpp:815
 msgid ""
 "Enable jogging/fast walking in town for Diablo and Hellfire. This option was "
 "introduced in the expansion."
 msgstr ""
+"Włączenie biegania/szybkiego chodzenia w mieście dla Diablo i Hellfire. "
+"Opcja ta została wprowadzona w rozszerzeniu."
```
Comment (qndel) [inline Translations/pl.po:6421]:
```suggestion
"Opcja ta została wprowadzona w dodatku."
```

File: Translations/pl.po [hunk 18]
```diff
@@ -6188,646 +6090,642 @@ msgstr "{:s} (wył.)"
...
 
 #: Source/options.cpp:827
 msgid "Adria will refill your mana when you visit her shop."
-msgstr ""
+msgstr "Adria uzupełni twoją manę, gdy odwiedzisz jej sklep."
 
 #: Source/options.cpp:828
 msgid "Auto Equip Weapons"
-msgstr ""
+msgstr "Automatycznie Wyposaż Broń"
```
Comment (qndel) [inline Translations/pl.po:6533]:
```suggestion
msgstr "Automatycznie Załóż Broń"
```

File: Translations/pl.po [hunk 19]
```diff
@@ -6188,646 +6090,642 @@ msgstr "{:s} (wył.)"
...
 
 #: Source/options.cpp:828
 msgid ""
 "Weapons will be automatically equipped on pickup or purchase if enabled."
 msgstr ""
+"Broń zostanie automatycznie wyposażona przy podnoszeniu lub zakupie, jeśli "
```
Comment (qndel) [inline Translations/pl.po:6539]:
```suggestion
"Broń zostanie automatycznie założona przy podnoszeniu lub zakupie, jeśli "
```

File: Translations/pl.po [hunk 20]
```diff
@@ -6188,646 +6090,642 @@ msgstr "{:s} (wył.)"
...
+"dana opcja jest włączona."
 
 #: Source/options.cpp:829
 msgid "Auto Equip Armor"
-msgstr ""
+msgstr "Automatycznie Wyposaż Pancerz"
```
Comment (qndel) [inline Translations/pl.po:6544]:
```suggestion
msgstr "Automatycznie Załóż Pancerz"
```

File: Translations/pl.po [hunk 21]
```diff
@@ -6188,646 +6090,642 @@ msgstr "{:s} (wył.)"
...
 
 #: Source/options.cpp:829
 msgid "Armor will be automatically equipped on pickup or purchase if enabled."
 msgstr ""
+"Pancerze będą automatycznie wyposażane przy podnoszeniu lub zakupie, jeśli "
```
Comment (qndel) [inline Translations/pl.po:6549]:
```suggestion
"Pancerze będą automatycznie zakładane przy podnoszeniu lub zakupie, jeśli "
```

File: Translations/pl.po [hunk 22]
```diff
@@ -6188,646 +6090,642 @@ msgstr "{:s} (wył.)"
...
+"dana opcja jest włączona."
 
 #: Source/options.cpp:830
 msgid "Auto Equip Helms"
-msgstr ""
+msgstr "Automatycznie Wyposaż Hełmy"
```
Comment (qndel) [inline Translations/pl.po:6554]:
```suggestion
msgstr "Automatycznie Załóż Hełmy"
```

File: Translations/pl.po [hunk 23]
```diff
@@ -6188,646 +6090,642 @@ msgstr "{:s} (wył.)"
...
 
 #: Source/options.cpp:830
 msgid "Helms will be automatically equipped on pickup or purchase if enabled."
 msgstr ""
+"Hełmy będą automatycznie wyposażane przy podnoszeniu lub zakupie, jeśli dana "
```
Comment (qndel) [inline Translations/pl.po:6559]:
```suggestion
"Hełmy będą automatycznie zakładane przy podnoszeniu lub zakupie, jeśli dana "
```

File: Translations/pl.po [hunk 24]
```diff
@@ -6188,646 +6090,642 @@ msgstr "{:s} (wył.)"
...
+"opcja jest włączona."
 
 #: Source/options.cpp:831
 msgid "Auto Equip Shields"
-msgstr ""
+msgstr "Automatycznie Wyposaż Tarczę"
```
Comment (qndel) [inline Translations/pl.po:6564]:
```suggestion
msgstr "Automatycznie Załóż Tarcze"
```

File: Translations/pl.po [hunk 25]
```diff
@@ -6188,646 +6090,642 @@ msgstr "{:s} (wył.)"
...
 
 #: Source/options.cpp:831
 msgid ""
 "Shields will be automatically equipped on pickup or purchase if enabled."
 msgstr ""
+"Tarcze będą automatycznie wyposażane przy podnoszeniu lub zakupie, jeśli "
```
Comment (qndel) [inline Translations/pl.po:6570]:
```suggestion
"Tarcze będą automatycznie zakładane przy podnoszeniu lub zakupie, jeśli "
```

File: Translations/pl.po [hunk 26]
```diff
@@ -6188,646 +6090,642 @@ msgstr "{:s} (wył.)"
...
+"dana opcja jest włączona."
 
 #: Source/options.cpp:832
 msgid "Auto Equip Jewelry"
-msgstr ""
+msgstr "Automatycznie Wyposaż Biżuterię"
```
Comment (qndel) [inline Translations/pl.po:6575]:
```suggestion
msgstr "Automatycznie Załóż Biżuterię"
```

File: Translations/pl.po [hunk 27]
```diff
@@ -6188,646 +6090,642 @@ msgstr "{:s} (wył.)"
...
 
 #: Source/options.cpp:832
 msgid ""
 "Jewelry will be automatically equipped on pickup or purchase if enabled."
 msgstr ""
+"Biżuteria zostanie automatycznie wyposażona przy podnoszeniu lub zakupie, "
```
Comment (qndel) [inline Translations/pl.po:6581]:
```suggestion
"Biżuteria zostanie automatycznie założona przy podnoszeniu lub zakupie, "
```

File: Translations/pl.po [hunk 28]
```diff
@@ -6188,646 +6090,642 @@ msgstr "{:s} (wył.)"
...
 
 #: Source/options.cpp:835
 msgid "Auto Refill Belt"
-msgstr ""
+msgstr "Automatycznie uzupełnij pas"
 
 #: Source/options.cpp:835
 msgid "Refill belt from inventory when belt item is consumed."
-msgstr ""
+msgstr "Uzupełnij pasek z ekwipunku, gdy element pasa zostanie zużyty."
```
Comment (qndel) [inline Translations/pl.po:6610]:
```suggestion
msgstr "Uzupełnij pasek z ekwipunku, gdy przedmiot w nim zostanie zużyty."
```

File: Translations/pl.po [hunk 29]
```diff
@@ -6188,646 +6090,642 @@ msgstr "{:s} (wył.)"
...
 
 #: Source/options.cpp:895
 msgid "Network Settings"
-msgstr ""
+msgstr "Ustawienia Sieciowe"
 
 #: Source/options.cpp:907
 msgid "Chat"
-msgstr ""
+msgstr "Chat"
```
Comment (qndel) [inline Translations/pl.po:6704]:
```suggestion
msgstr "Czat"
```

File: Translations/pl.po [hunk 30]
```diff
@@ -6188,646 +6090,642 @@ msgstr "{:s} (wył.)"
...
-msgstr ""
+msgstr "Wyłącz kaleczące kapliczki"
 
 #: Source/options.cpp:836
 msgid ""
 "When enabled Cauldrons, Fascinating Shrines, Goat Shrines, Ornate Shrines "
 "and Sacred Shrines are not able to be clicked on and labeled as disabled."
 msgstr ""
+"Po włączeniu opcji Kociołki, Fascynujące Kapliczki, Kozie Kapliczki, "
+"Zdobione Kapliczki i Święte Kapliczki nie można klikać i są one oznaczane "
```
Comment (qndel) [inline Translations/pl.po:6622]:
```suggestion
"Zdobione Kapliczki i Święte Kapliczki nie mogą zostać kliknięte i są one oznaczane "
```

File: Translations/pl.po [hunk 31]
```diff
@@ -6188,646 +6090,642 @@ msgstr "{:s} (wył.)"
...
 msgid "Friendly Fire"
-msgstr ""
+msgstr "Przyjacielski ogień"
 
 #: Source/options.cpp:819
 msgid ""
 "Allow arrow/spell damage between players in multiplayer even when the "
 "friendly mode is on."
 msgstr ""
+"Umożliwienie zadawania obrażeń od strzałów/czarów między graczami w trybie "
```
Comment (qndel) [inline Translations/pl.po:6459]:
```suggestion
"Umożliwienie zadawania obrażeń od strzał/czarów między graczami w trybie "
```

**Pull request merged**